vm/vm.go

```diff
@@ -45,6 +48,12 @@ import (
 	avasync "github.com/ava-labs/avalanchego/x/sync"
 )
 
+const (
+	blockDB   = "blockdb"
+	stateDB   = "statedb"
+	vmDataDir = "vm"
```

Comment:
Open to considerations for this directory's name. `Controller` as a name seems weird to me so left it as `vm`.